DigitalAssetManagement/Digital Asset Management/AppInsightsHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DigitalAssetManagementTemplate
+{
+    class AppInsightsHelper
```

Comment:
Add some comment here. e.g. Use this class as a central place to log any app insights events.